_stbt/core.py

```diff
@@ -1154,6 +1154,16 @@ def ocr(self, frame=None, region=Region.ALL,
         if lang is None:
             lang = get_config("ocr", "lang", "eng")
 
+        if tesseract_user_words is not None and not isinstance(
+                tesseract_user_words, list):
+            raise TypeError(
+                "stbt.ocr: tesseract_user_words must be a list of strings")
+
+        if tesseract_user_patterns is not None and not isinstance(
+                tesseract_user_patterns, list):
+            raise TypeError(
+                "stbt.ocr: tesseract_user_patterns must be a list of strings")
```

Comment:
I think being able to pass in a set or a tuple is useful.  Perhaps specifically blacklisting strings would be better rather than whitelisting lists?